packages/schemas/src/seeds/application.ts

```diff
@@ -27,6 +27,7 @@ export const buildDemoAppDataForTenant = (tenantId: string): Application => ({
   type: ApplicationType.SPA,
   oidcClientMetadata: { redirectUris: [], postLogoutRedirectUris: [] },
   customClientMetadata: {},
+  protectedAppConfigs: null,
```

Comment:
should we align with the `metadata` convention?
```suggestion
  protectedAppMetadata:{},
```